db/schema.rb

```diff
@@ -124,8 +124,9 @@
     t.string   "state"
     t.integer  "zip"
     t.integer  "code_school_id"
-    t.datetime "created_at",     null: false
-    t.datetime "updated_at",     null: false
+    t.datetime "created_at",       null: false
+    t.datetime "updated_at",       null: false
+    t.boolean  "provides_housing"
```

Comment:
Given the issue's statement:

> And when performing migration set a default state of False.

I would assume `false` is a reasonable default for any new record, though I agree it is not crystal clear.

Reply:
lol - I guess I just overlooked the basic requirements. Updating now.